custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -109,11 +108,9 @@ def get_person_case_structure(self):
                     'dob_known': 'no',
                     'first_name': self.patient_detail.first_name,
                     'last_name': self.patient_detail.last_name,
-                    'middle_name': self.patient_detail.middle_name,
                     'name': self.patient_detail.pname,
                     'nikshay_id': self.nikshay_id,
-                    'permanent_address_district_choice': self.patient_detail.Dtocode,
-                    'permanent_address_state_choice': self.patient_detail.scode,
+                    'person_id': 'FROM_NIKSHAY_' + self.nikshay_id,
```

Comment:
[12:27:12 PM] Kriti Mehrotra: hmm
[12:28:41 PM] Farid Rener: or person.person_id = "(from Nikshay) {nikshay_id}"
[12:28:52 PM] Farid Rener: like we have for "test ids"
[12:29:41 PM] Kriti Mehrotra: yeah, just some sort of prefix will help I think
[12:29:47 PM] Farid Rener: cool beans